.github/workflows/dead-links-check.yml

```diff
@@ -13,5 +13,5 @@ jobs:
       - name: Link Checker
         uses: lycheeverse/lychee-action@v1.8.0
         with:
-          args: " --suggest --verbose --no-progress './**/*.md' './**/*.html' './**/*.rst' --exclude-mail -a 429 --exclude-path *fin7/Resources/Step7/BOOSTWRITE-src/curl/README.md "
+          args: " --suggest --verbose --no-progress './**/*.md' './**/*.html' './**/*.rst' --exclude-mail -a 403,429 --exclude-path *fin7/Resources/Step7/BOOSTWRITE-src/curl/README.md "
```

Comment:
- `-- suggest`  adds wayback links
- `--verbose --no-progress` format of output
- *.md etc. targeting only selected files (not .c for example) `--include-verbatim` could add search inside md code blocks
-  `-a` treats http codes 403 and 429 as good  Bitdefender and about two others returns those (due to needed cookies and js) - maybe could be replaced with specific exclusion `--exclude` 
-  `--exclude-path` this one file is in UTF-16 (?bug?) link checker chrashes on this (very rarely even now after excluding )